src/contractTest/java/uk/gov/hmcts/reform/cdapi/provider/CommonDataApiProviderTest.java

```diff
@@ -143,10 +143,10 @@ private ListOfValueDto buildListOfValueDto(String categoryId, String key, String
         listOfValueDto1.setHintTextEn(null);
         listOfValueDto1.setValueCy(null);
         listOfValueDto1.setValueEn(value);
-        listOfValueDto1.setServiceId("BBA3");
         CategoryKey categoryKey = new CategoryKey();
         categoryKey.setKey(key);
         categoryKey.setCategoryKey(categoryId);
+        categoryKey.setServiceId("BBA3");
         listOfValueDto1.setCategoryKey(categoryKey);
         return listOfValueDto1;
     }
```

Comment:
I don't see integration tests here, but i have seen the existing integration tests and i dont see below scenarios covered. pls add test cases for below scenarios in integration tests:
1. request call with category id and service id with no child records
2. request call with category id and service id with child records

Reply:
Hi @rajkatla-hmcts Thanks for the above comment 
[src/contractTest/java/uk/gov/hmcts/reform/cdapi/provider/CommonDataApiProviderTest.java] is the PACT Test case .... Instead i have added the same above scenarios in the functional test cases please ... Thank you

Reply:
Hi @rajkatla-hmcts added integration test cases as well ...Thank you